lib/flutter_login.dart

```diff
@@ -331,6 +334,16 @@ class FlutterLogin extends StatefulWidget {
   /// Navigate back to the login screen after recovery of password.
   final bool navigateBackAfterRecovery;
 
+  /// Prefilled (ie. saved from previous session) value at startup for username
+  /// (Auth class calls username email, therefore we use savedEmail here aswell)
+  final String savedEmail;
+
+  /// Prefilled (ie. saved from previous session) value at startup for password
+  final String savedPassword;
+
+  /// Prefilled (ie. saved from previous session) value at startup for confirmation password
+  final String savedConfirmPassword;
```

Comment:
Why would we have a separate confirm password field stored from previous session? It would not different form the savePassword. Otherwise the method that saved it in the first place didn't check if the password was valid.

Reply:
@juliansteenbakker indeed you are right :-) 

I found even saving `savedConfirmPassword` a bit useless, but it was done for completeness (so all three `Auth` parameters can be set). But indeed: I can fill that from `savedPassword`.

I'll amend right now, so `savedPassword` is used for both (if passed of course).

Reply:
Added in last commit! I can of course squash at the end, after other comments/reviews, to keep commit history cleaner if you prefeer!